CIP-0001/README.md

```diff
@@ -12,180 +10,120 @@ License: CC-BY-4.0
-| Abstract | A short (\~200 word) description of the technical issue being addressed. |
-| Motivation | The motivation is critical for CIPs that want to change the Cardano protocol. It should clearly explain why the existing protocol is inadequate to address the problem that the CIP solves. |
-| Specification | The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Cardano platforms. |
-| Rationale | The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during discussion.|
-| Backwards compatibility | All CIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The CIP must explain how the author proposes to deal with these incompatibilities.|
-| Path to Active | A reference implementation, observable metrics or anything showing the acceptance of the proposal in the community. It must be completed before any CIP is given status "Active", but it need not be completed before the CIP is accepted. It is better to finish the specification and rationale first and reach consensus on it before writing any code. |
-| Copyright | The CIP must be explicitly licensed under acceptable copyright terms ([see below](#CIP-licensing)).|
+## Specification
 
-### CIP Workflow
+### Types
 
-A CIP begins as an idea about how to improve the Cardano network or its surrounding processes, with one or more individuals who are willing to propose and discuss the idea with the Cardano community. After initial discussion and feedback, the idea is then formalized into a properly formatted proposal and submitted as a pull request to the CIP repository. The resulting new Draft CIP is then publicly processed and progressed as follows:  
+Type          | Description
+---           | ---
+Core          | A core CIP describes a change affecting the Cardano blockchain's core components. For instance, a change in the ledger rules, a modification of the on-the-wire binary format or adjustments to the rewards and incentives strategy.
+Process       | A process defines a common standard to follow to achieve a certain degree of interoperability between various actors. It can be a formal document describing a specific procedure or a set of recommendations/guidelines on one particular topic.
+Informational | An informational CIP documents a particular design decision. These are often written a posteriori to address a gap in understanding and knowledge sharing.
 
-![CIP Flow](CIP_Flow.png)  
+### Structure
 
+Each CIP should have the following parts:
```

Comment:
must? you say "must" in some other contexts, so I would interpret this as saying they are optional, and I'm not sure if that's what you mean